### PR TITLE
Components

### PR DESCRIPTION
**Pull request by yahyaelganyni1**
## components branch

in this branch, I added the calculator components to the react app and style it

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+function App() {
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
+}
+
+export default App;
```
Comment (Emmyn5600) [inline src/App.js]: - [x] The App.js component should be class-based.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+
```
Comment (Emmyn5600) [inline src/components/Calculator.js:3]: - [ ] Don't disable eslint errors, kindly fix them without disabling them.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

### Hi, @yahyaelganyni1  

Great work on implementing the project requirements, however, there are some issues to be solved before you get approved but you are almost there:muscle:
 
### Changes requested.
- _Check the comment under the review too_

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by yahyaelganyni1** [comment]
Untitled!

for some reason, I can't resolve this problem in the esLent. so that's why I added the ` /* eslint-disable */ `
`

**Review by AshakaE — APPROVED** [review]
### Hi @yahyaelganyni1  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**